src/main/webapp/projectDocument/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ProjectDocument from './ProjectDocument';
+import '../index.css';
+
+
+ ReactDOM.render(
+    <ProjectDocument
+        projectKey = {component.projectKey}
+        attachDocumentsUrl = {component.attachDocumentsUrl}
+        attachedDocumentsUrl = {component.attachedDocumentsUrl}
+        approveDocumentUrl = {component.approveDocumentUrl}
+        rejectDocumentUrl = {component.rejectDocumentUrl}
+        isAdmin = {component.isAdmin}
+        component = {component}
+        downloadDocumentUrl = {component.downloadDocumentUrl}
+        user = {component.user}
```

Comment:
See previous comments about this. We should probably be fetching the user before this is rendered and passing that in.